src/main/java/build/buildfarm/cas/cfc/CASFileCache.java

```diff
@@ -2585,6 +2602,26 @@ private CancellableOutputStream putOrReference(
     }
   }
 
+  private void deleteExpiredKey(String expiredKey) throws IOException {
```

Comment:
Just take the path here - you don't concern yourself with the key except to getPath

Reply:
fixed